src/app.ts

```diff
@@ -99,6 +99,45 @@ async function app() {
 
   logger.info(`Starting ${getVersion()}`);
 
+  logger.info('Arguments:');
+  if (programOptions.logLevel) {
+    logger.info(`  --log-level ${programOptions.logLevel}`);
+  }
+  if (programOptions.dependency) {
+    logger.info(`  --dependency ${programOptions.dependency}`);
+  }
+  if (programOptions.style) {
+    logger.info(`  --style ${programOptions.style}`);
+  }
+  if (programOptions.fshingTrip) {
+    logger.info('  --fshing-trip');
+  }
+  if (programOptions.installedSushi) {
+    logger.info('  --installed-sushi');
+  }
+  if (programOptions.indent) {
+    logger.info('  --indent');
+  }
+  if (programOptions.installedSushi) {
+    logger.info('  --installed-sushi');
+  }
+  if (programOptions.metaProfile) {
+    logger.info(`  --meta-profile ${programOptions.metaProfile}`);
+  }
+  if (programOptions.aliasFile) {
+    logger.info(`  --alias-file ${programOptions.aliasFile}`);
+  }
+  if (programOptions.alias) {
```

Comment:
The logic for this one actually needs to be flipped. When `programOptions.alias` is `true`, we are using aliases and we have _not_ used the `--no-alias` flag. The value of `programOptions.alias` gets flipped to `false` when you do use `--no-alias`. So we want to only print the option information when we used `--no-alias`, so instead I think we need to check `if(!programOptions.alias)`.